App.tsx

```diff
@@ -33,12 +42,29 @@ const App = (): JSX.Element => {
     }
   }, [loadingFonts]);
 
+  useEffect(() => {
+    Analytics.setDebugModeEnabled(true);
+  }, []);
```

Comment:
Is this right for production?

Reply:
@luizeboli esse debugMode é uma feature do Firebase Analytics que habilita uma timeline de eventos... O nome é confuso mesmo rsrs

Reply:
Ah tá, sucesso hehe